pkg/util/utils_windows.go

```diff
@@ -30,8 +30,8 @@ func GetRootlessPauseProcessPidPath() (string, error) {
 	return "", fmt.Errorf("GetRootlessPauseProcessPidPath: %w", errNotImplemented)
 }
 
-// GetRuntimeDir returns the runtime directory
-func GetRuntimeDir() (string, error) {
+// GetRootlessRuntimeDir returns the runtime directory
+func GetRootlessRuntimeDir() (string, error) {
```

Comment:
Should this also be consolidated with c/storage? … but it looks so different…

Reply:
that was done in the previous implementation, but we decided for the storage version to not return the empty string for `!IsRootless()` as it was a breaking change, so the check is left here in Podman